src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -51,7 +51,7 @@ def validate_import_key(key):
         if not isinstance(key, str):
             logger.warning("Ignoring invalid key '%s'. Key must be a string.", key)
             return False
-        if key == '.' or key == '..' or '%' in key:
+        if key in (".", "..") or "%" in key:
```

Comment:
I am not sure why `ruff` changes `'` to `"`.

Reply:
Black?

Reply:
@atombrella, could you explain in detail?

Reply:
My guess is that Ruff would format according to Black. In the documentation for Ruff, it's described as a "drop-in replacement for [Black](https://pypi.org/project/black/". https://docs.astral.sh/ruff/formatter/ But I don't know for sure, because it doesn't seem to change the quotes consistently in this change.

https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings